boards/iotlab-a8-m3/Makefile.include

```diff
@@ -1,3 +1,7 @@
 USEMODULE += boards_common_iotlab
 
+DEBUG_ADAPTER ?= iotlab
```

Comment:
Those 2 lines could be moved in the `common/iotlab/Makefile.include` file, same for the iotlab-la8-m3.

Reply:
done